horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/ActionServiceImpl.java

```diff
@@ -71,6 +72,9 @@ public class ActionServiceImpl implements ActionService {
    @Inject
    EncryptionManager encryptionManager;
 
+   @Inject
+   TestServiceImpl testService;
```

Comment:
Minor nitpick, This could be the api, which allows CDI to decide which impl to inject. IDK if we are ever going to have multiple impls, so prob not neccessary

Reply:
Unfortunately the getTestForUpdate method is not exposed on the api.